qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -106,12 +111,18 @@ public void test010Install() {
     }
 
     /**
-     * Check that the /_xpack API endpoint's presence is correct for the type of distribution being tested.
+     * Check that security is enabled
      */
-    public void test011PresenceOfXpack() throws Exception {
-        waitForElasticsearch(installation);
-        final int statusCode = Request.Get("http://localhost:9200/_xpack").execute().returnResponse().getStatusLine().getStatusCode();
+    public void test011SecurityEnabledStatus() throws Exception {
+        waitForElasticsearch(installation, USERNAME, PASSWORD);
+        final int statusCode = ServerUtils.makeRequestAndGetStatus(Request.Get("http://localhost:9200"), USERNAME, PASSWORD, null);
```

Comment:
This is not quite the best check to validate security because credentials are ignored when security is disabled.

I would call `_xpack/usages` (and assert the output) or use wrong credentials, or both.

Reply:
But I won't dwell on it anymore, it's likely to be changed soon anyway.

Reply:
changed this to expect 401 with wrong credentials